wetterdienst/provider/dwd/mosmix/access.py

```diff
@@ -88,9 +88,7 @@ def read(self, url: str):
         kml = self.fetch(url)
 
         log.info("Parsing KML data")
-        # TODO: Check if XML parsing performance can be improved by using libxml2.
-        tree = parse(BytesIO(kml), parser=XMLParser(huge_tree=True))  # noqa: S320
-        self.root = root = tree.getroot()
+        self.iter_elems = iterparse(BytesIO(kml), events=("start", "end"))
```

Comment:
Maybe to satisfy the linter you could add the argument `resolve_entities=False` like
```suggestion
        self.iter_elems = iterparse(BytesIO(kml), events=("start", "end"), resolve_entities=False)
```